File: 8KYU/Total Amount of Points/solution.py
def points(games):
    parsed_scores =  [[int(game.split(':')[0]), int(game.split(':')[1])] for game in games]
    res = 0
    for score in parsed_scores:
        if score[0] == score[1]:
            res+= 1
        elif score[0] > score[1]:
            res+=3
    return res

File: 8KYU/Total Amount of Points/solution.go
package kata

import (
	"strconv"
	"strings"
)

func ParseScore(game string) (int, int) {
	score := strings.Split(game, ":")
	home, _ := strconv.Atoi(score[0])
	away, _ := strconv.Atoi(score[1])
	return home, away
}

func Points(games []string) int {
	total := 0
	for _, game := range games {
		home, away := ParseScore(game)
		if home > away {
			total = total + 3
		} else if home == away {
			total = total + 1
		}
	}
	return total
}
